app/Helpers/MailSend.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Helpers;
+
+
+class MailSend
+{
+    /**
+     * Send email to user
+     * @param $view
+     * @param $data
+     * @param $profileEmail
+     * @param $profileName
+     * @param $subject
+     * @return bool
+     */
+    public static function send($view, $data, $profileEmail, $profileName, $subject)
```

Comment:
Send in `$profile` and read what you need from there, not `$profileName` and `$profileEmail` separately